src/test/java/org/gridsuite/shortcircuit/server/ShortCircuitAnalysisControllerTest.java

```diff
@@ -381,6 +396,36 @@ public void runTest() throws Exception {
             List<org.gridsuite.shortcircuit.server.dto.FaultResult> faultResultsPageDto1Full = faultResultsReader.readValue(faultResultsPageNode1.get("content"));
             assertPagedFaultResultsEquals(ShortCircuitAnalysisResultMock.RESULT_SORTED_PAGE_1, faultResultsPageDto1Full);
 
+            // export zipped csv result
+            result = mockMvc.perform(post(
+                            "/" + VERSION + "/results/{resultUuid}/csv", RESULT_UUID)
+                            .contentType(MediaType.APPLICATION_JSON)
+                            .content(CSV_HEADERS))
+                    .andExpectAll(status().isOk(), content().contentType(MediaType.APPLICATION_OCTET_STREAM))
+                    .andReturn();
+            byte[] zippedResult = result.getResponse().getContentAsByteArray();
+            try (ByteArrayInputStream byteArrayInputStream = new ByteArrayInputStream(zippedResult);
```

Comment:
Should extract below unzip code as we did in sensitivity to make test code more clean,
see https://github.com/gridsuite/sensitivity-analysis-server/pull/60/files#diff-24b2d5e07dcacf5c505fffd71b1d39006c833f6b41ccf1cfa5e0cb8b151839faR29

Reply:
done